ripd/rip_cli.c

```diff
@@ -915,7 +915,7 @@ DEFPY (no_ip_rip_authentication_string,
        "Authentication string\n"
        "Authentication string\n")
 {
-	nb_cli_enqueue_change(vty, "./authentication-password", NB_OP_MODIFY,
+	nb_cli_enqueue_change(vty, "./authentication-password", NB_OP_DELETE,
 			      NULL);
```

Comment:
by seeing that change, does that mean that for instance, a combinated vty command like rip_default_information_originate command, we have to set NB_OP_DELETE when the no form is called, and NB_OP_MODIFY when the positive form is called ? I mean, if there is some fixes to be done here, this can be part of a separate ticket.

Reply:
`NB_OP_MODIFY` will create the data node if it doesn't exist, so this works fine.